ui/components/app/assets/auto-detect-token/auto-detect-token-modal.tsx

```diff
@@ -101,13 +100,9 @@ function AutoDetectTokenModal({
           >
             <img src="/images/wallet-alpha.png" />
           </Box>
-          <Text
-            variant={TextVariant.bodyMd}
-            textAlign={TextAlign.Justify}
-            padding={0}
-          >
+          <Text variant={TextVariant.bodyMd} padding={0}>
```

Comment:
non-blocking: `Text` component has no padding by default I don't believe padding={0} is needed here
```suggestion
          <Text variant={TextVariant.bodyMd}>
```